mmdeploy/codebase/mmcls/deploy/classification_model.py

```diff
@@ -155,8 +155,10 @@ def get_classes_from_config(model_cfg: Union[str, mmcv.Config]):
     model_cfg = load_config(model_cfg)[0]
     data_cfg = model_cfg.data
 
-    def _get_class_names(dataset_type):
+    def _get_class_names(dataset_type: str):
         dataset = data_cfg[dataset_type]
+        if not dataset:
```

Comment:
Errrr, I means `dataset.type not in module_dict == True`.
This might happen when the user trains a model in a device and deployment it in another one.